src/ros_filter.cpp

```diff
@@ -706,6 +743,9 @@ namespace RobotLocalization
     nhLocal_.param("smooth_lagged_data", smoothLaggedData_, false);
     nhLocal_.param("history_length", historyLength_, 0.0);
 
+    // Wether we reset filter on jump back in time
+    nhLocal_.param("reset_on_jump_back_in_time", resetOnJumpBackInTime_, false);
```

Comment:
This parameter name says what it does, but I wonder if we could think of something less verbose.